content/en/users/im/dashboard/_index.md

```diff
@@ -0,0 +1,181 @@
+---
+title: "IM Dashboard"
```

Comment:
Bad title, see my comment on the CLI page's title. I'd use "Infrastructure Dashboard" on the left, "Dashboard" as title, and "The dashboard of the Infrastructure Manager".

Reply:
changed.